pages/windows/driverquery.md

```diff
@@ -0,0 +1,31 @@
+# driverquery
+
+> Display information about installed device drivers.
+
+- Display a list of all installed device drivers:
+
+`driverquery`
+
+- Display a list of drivers in the specified format:
+
+`driverquery /fo {{table|list|csv}}`
+
+- Display a list of drivers with a column for if they are signed:
+
+`driverquery /si`
+
+- Exclude the header in the output list:
+
+`driverquery /nh`
+
+- Display a list of drivers for a remote machine:
+
+`driverquery /s {{remote_name}} /u {{username}} /p {{password}}`
```

Comment:
I think hostname is better than remote_name. The word remote sounds like git remote.